list_books.rb

```diff
@@ -3,8 +3,8 @@
 def list_books(books)
   puts 'All books:'
 
-  books.each_with_index do |book, index|
-    puts "#{index + 1}-> #{book.title.capitalize} by #{book.author.capitalize}"
+  books.each_with_index do |book, _index|
+    puts "#{book.title.capitalize}, Author:#{book.author.capitalize}"
```

Comment:
- [ ] Good job implementing the changes given by the second reviewer now I can create a person and generate rentals but please as mentioned by the first reviewer you should follow the given design and add the title and author header but you add only the `Author` header 🙃 🙃 

Your design

<img width="529" alt="Screenshot 2023-04-13 at 21 33 00" src="https://user-images.githubusercontent.com/74501523/231852440-3abc61c3-1c11-4242-a623-1a8e74c19bec.png">

Given design

![231775972-9dbe1f4e-3b95-4884-b82f-1b4ed9d457ca](https://user-images.githubusercontent.com/74501523/231852757-b12ab0a0-0fd6-4ddf-a9d2-b639f0efd8e4.png)